CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 3.3.2 (September 22, 2023)
+
+IMPROVEMENTS:
+* Support on-demand connection init in the provider. This behavior is controlled with `on_demand_connection` flag and is useful is NSX manager is not available at the time of plan/apply ([#948](https://github.com/vmware/terraform-provider-nsxt/pull/948))
+* `resource/nsxt_policy_tier1_gateway`: Support `type` argument. This argument helps with auto-configuring route advertisiements and provides the user experience thai is consistent with UI on VMC ([#909](https://github.com/vmware/terraform-provider-nsxt/pull/909))
+* Improve debug logging by dumping NSX API requests and responses when `TF_LOG_PROVIDER_NSX_HTTP` env variable is set ([#963](https://github.com/vmware/terraform-provider-nsxt/pull/963))
+
+BUG FIXES:
+* `resource/nsxt_policy_security_policy`, `resource/nsxt_policy_gateway_policy`, `resource/nsxt_policy_ids_policy`: Fix rule ordering issue by auto-assigning `sequence_number`. ([#967](https://github.com/vmware/terraform-provider-nsxt/pull/967))
+* `resource/nsxt_policy_group`: Fix `group_type` assignment on VMC by using `node/version` API to determine underlying NSX version ([#970](https://github.com/vmware/terraform-provider-nsxt/pull/970))
+* `resource/nsxt_nat_rule`: Fix the resource, that was broken on recent NSX versions, by replacing `nat_pass` property, which was removed from platform, with `firewall_match` ([#950](https://github.com/vmware/terraform-provider-nsxt/pull/950))
```

Comment:
suggestion: ensure compatibility with NSX 4.1.0 and above by replacing removed 'nat_pass' property with 'firewall_match'